storybookjs/storybook
# CLI: Add a `--webpack-stats-json` flag

Issue: #14139

## What I did

Add a `--webpack-stats-json` flag (for build and dev) that allows you to pass a directory to dump webpack's stats info. Change `--smoke-test` to no longer do that implicitly.

## How to test

- Is this testable with Jest or Chromatic screenshots?

Not that I know of, correct me if I am wrong.

It is implicitly tested by the packtracker task.

- Does this need an update to the documentation?

Updated CLI docs.

I'm not sure if we should document this in `MIGRATION.md` or not.

If your answer is yes to any of these, please make sure to include it in your PR.
